app/routers/event.py

```diff
@@ -64,17 +70,18 @@ async def create_new_event(request: Request,
     location = data['location']
     vc_link = data['vc_link']
     category_id = data.get('category_id')
-
     invited_emails = get_invited_emails(data['invited'])
     uninvited_contacts = get_uninvited_regular_emails(session, owner_id,
                                                       title, invited_emails)
+    shared_list = extract_shared_list_from_data(data, session)
```

Comment:
Avoid using `list` or `data` as names - these won't tell the reader what you're trying to accomplish